src/index.scss

```diff
@@ -65,7 +67,7 @@ $white: #fff;
   }
   .main-nav {
     .nav-link {
-      padding: 1.125rem 1rem;
+      padding: 0 1rem;
```

Comment:
Let's try to keep our nav links the full height of the header. To this end, we should probably prefer `height` or `min-height` instead of padding top and bottom on line 48 of this file.

Reply:
Noted, added `height: 3.75rem` to .site-header-desktop.